### DISCUSSION TITLE
Same number of layers but different parameter count.

### DISCUSSION THREAD

**Discussion by PraveenMNaik** [ultralytics/yolov5#10158] [Q&A]
@glenn-jocher , the yolov5n and yolov5s models have same number of layers but varies with number of parameters, what is the reason?. Is it only because of the way we used depth and width multiples? How the parameters are selected?

**Comment by glenn-jocher** [comment]
@PraveenMNaik yes that's correct, n to s scales horizontally (channels) but not vertically (C3 repeats).
